srm/__manifest__.py

```diff
@@ -0,0 +1,29 @@
+# Copyright 2022 Telmo Santos <telmo.santos@camptocamp.com>
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+
+{
+    "name": "SRM",
+    "version": "14.0.1.0.0",
+    "author": "Camptocamp, Odoo Community Association (OCA)",
+    "summary": "Use CRM model for suppliers",
+    "license": "AGPL-3",
+    "category": "CRM",
+    "depends": [
+        "account",
+        "crm",
+        "sale_crm",
+        "purchase",
+        "crm_enterprise",
```

Comment:
```suggestion
```
You can't add this proprietary module in an OCA module.

Reply:
You're right.